index.html

```diff
@@ -0,0 +1,158 @@
+        <section class="second-section">
+            <h2 class="main-p-text">Main Program</h2>
+            <span class="bar-gold"></span>
+            <div class="second-part">
+            <div class="parts">
+                <div class="parts-img-con">
+                <img src="./assets/icons/program_icon_01.png" alt="" class="parts-photo">
+                </div>
+                
+                <h3 class="activity-bar">Speech</h3>
+                <p>Listen to Leaders of the SAARC countries talking about ways to find solution to the economic growth, welfare and cultural development.</p>
+                
+            </div>
+            <div class="parts">
+                <div class="parts-img-con">
+                <img src="./assets/icons/program_icon_02.png" alt="" class="parts-photo">
+                </div>
+                
+                <h3 class="activity-bar">Speech</h3>
+                <p>Listen to Leaders of the SAARC countries talking about ways to find solution to the economic growth, welfare and cultural development.</p>
+                
+            </div>
+            <div class="parts">
+                <div class="parts-img-con">
+                <img src="./assets/icons/program_icon_03.png" alt="" class="parts-photo">
+                </div>
+                
+                <h3 class="activity-bar">Speech</h3>
+                <p>Listen to Leaders of the SAARC countries talking about ways to find solution to the economic growth, welfare and cultural development.</p>
+                
+            </div>
+            <div class="parts">
+                <div class="parts-img-con">
+                <img src="./assets/icons/program_icon_04.png" alt="" class="parts-photo">
+                </div>
+                
+                <h3 class="activity-bar">Speech</h3>
+                <p>Listen to Leaders of the SAARC countries talking about ways to find solution to the economic growth, welfare and cultural development.</p>
+               
+            </div>
+            <div class="parts">
+                <div class="parts-img-con">
+                <img src="./assets/icons/program_icon_05.png" alt="" class="parts-photo">
+                </div>
+                
+                <h3 class="activity-bar">Speech</h3>
+                <p>Listen to Leaders of the SAARC countries talking about ways to find solution to the economic growth, welfare and cultural development.</p>
+                
+            </div>
+            </div>
+            <button class="join-button">Join SAARC Summit 2022</button>
+            <a href="#" class="parts-link">SEE THE WHOLE PROGRAM</a>
+        </section>
+        <!--Third Section-->
+        <section class="third-part">
+            <h2 class="featured-header">Featured Speakers</h2>
+            <span class="bar-gold"></span>
+            <article class="third-section"></article>
+            
+        </section>
+
+        <section class="partner-section">
```

Comment:
The featured speakers' sections should have 6 individual cards but you have only 4. Please fix it 🙏